engine/CMakeLists.txt

```diff
@@ -1298,7 +1298,7 @@ IF (CMAKE_SYSTEM_NAME STREQUAL Windows)
     INSTALL(TARGETS vegastrike-engine COMPONENT vegastrike_engine RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
     INSTALL(PROGRAMS "${CMAKE_CURRENT_BINARY_DIR}/${CPACK_BUILD_CONFIG}/vegastrike-engine.exe" RENAME vegastrike.exe DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT vegastrike_alias)
 ELSEIF (CMAKE_SYSTEM_NAME STREQUAL Darwin)
-    INSTALL(TARGETS vegastrike-engine COMPONENT vegastrike_engine BUNDLE DESTINATION "${CMAKE_OSX_APPLICATIONS_DIR}/VegaStrike")
+    INSTALL(TARGETS vegastrike-engine DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT vegastrike_engine BUNDLE DESTINATION "/Applications/VegaStrike")
```

Comment:
We should probably make this a variable so users could override for a personal install. There's also an envelope var for it.

Reply:
An envelope var?